rand/guid/README.md

```diff
@@ -0,0 +1,33 @@
+# rand/guid
+
+The benchmark compares methods of generating a globally unique string identifier (GUID).
```

Comment:
@dim I'd still suggest to tell the difference between random (I assume purely random) and sorted (is that smth about like counter per host/process?)

Reply:
yep

Reply:
I meant right in readme